packages/pds/src/db/migrations/20230124T230923517Z-user-partitioned-cids.ts

```diff
@@ -0,0 +1,84 @@
+import { Kysely, sql } from 'kysely'
+import { Dialect } from '..'
+
+export async function up(db: Kysely<any>, dialect: Dialect): Promise<void> {
+  const binaryDatatype = dialect === 'sqlite' ? 'blob' : sql`bytea`
+
+  await db.schema
+    .createTable('ipld_block_temp')
+    .addColumn('cid', 'varchar', (col) => col.notNull())
+    .addColumn('creator', 'varchar', (col) => col.notNull())
+    .addColumn('size', 'integer', (col) => col.notNull())
+    .addColumn('content', binaryDatatype, (col) => col.notNull())
+    .addColumn('indexedAt', 'varchar', (col) => col.notNull())
+    .addPrimaryKeyConstraint('ipld_block_with_creator_pkey', ['cid', 'creator'])
```

Comment:
If we want it to be cheap to enumerate blocks by creator, might suggest flipping around this index to `['creator', 'cid']`.

Reply:
nice i like it 👍